handler/pair.go

```diff
@@ -0,0 +1,47 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"path"
+
+	"github.com/gorilla/mux"
+
+	"github.com/cloudradar-monitoring/plexus/api"
+)
+
+// Pair godoc
+// @Summary Gets the pairing code info.
+// @Tags session
+// @Produce application/json
+// @Param code path string true "code"
+// @Security BasicAuth
+// @Success 200 {object} api.URLResponse
+// @Failure 401 {object} api.Error
+// @Failure 404 {object} api.Error
+// @Router /session/{code} [get]
+func (h *Handler) Pair(rw http.ResponseWriter, r *http.Request) {
+	h.lock.Lock()
+	defer h.lock.Unlock()
+	id, ok := h.codes[mux.Vars(r)["id"]]
```

Comment:
I think it would be better iterating over the session map and finding a session that has the pairing code. This extra map just adds complexity. The session map already will be cleaned accordingly, the codes map will just grow endlessly.

Reply:
When deleting sessions we can delete the code as well.
But if you think iterating is better okay ill change it.